webapp/src/styles/index.ts

```diff
@@ -199,11 +129,12 @@ const themeMainLight = createTheme({
 
 // Dark Theme - https://www.realtimecolors.com/?colors=dbe5f5-070e18-8ca9de-282687-6350ce&fonts=Inter-Inter
```

Comment:
Let's remove this link, this was used to generate the colors for the previous theme and is no longer necessary

Reply:
Removed